source/docs/api-reference/api-usage/swerve/swerve-overview.rst

```diff
@@ -0,0 +1,66 @@
+Swerve Overview
+===============
+
+New in 2024 is a high performance swerve framework. This framework simplifies the boilerplate necessary for swerve and maximizes performance.
+
+.. important:: This section focuses on utilizing the Swerve API and configuring it correctly. Tuner X supports a swerve project creator that greatly simplifies this process and removes common error cases.
+
+- Small API surface, easily debuggable
+
+  * Build robot characteristics using ``SwerveModuleConstants`` (Java) and ``SwerveDrivetrainConstants``.
+  * Integrates cleanly into WPILib `commandbased <https://docs.wpilib.org/en/stable/docs/software/commandbased/index.html>`__ framework via ``CommandSwerveDrivetrain`` (Java).
```

Comment:
Note that `CommandSwerveDrivetrain` only exists in the example project, it's not part of the native API. We can't depend on the NewCommands vendordep in our API.